app/assets/javascripts/components/overview/milestones.jsx

```diff
@@ -4,6 +4,7 @@ import createReactClass from 'create-react-class';
 import { filter } from 'lodash-es';
 
 import CourseDateUtils from '../../utils/course_date_utils';
+import DateCalculator from '../../utils/date_calculator';
```

Comment:
Is this still needed?

Reply:
Nope,I forgot to remove the DateCalculator. I'll go ahead and remove it now, sorry about that.